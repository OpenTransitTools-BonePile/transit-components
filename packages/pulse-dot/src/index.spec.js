import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import { PulseDot, PulseCircle, PulseRing } from ".";

describe("Input", () => {
  test("renders correctly", () => {
    const tree = renderer
      .create(
        <PulseDot>
          <PulseCircle></PulseCircle>
          <PulseRing></PulseRing>
        </PulseDot>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
