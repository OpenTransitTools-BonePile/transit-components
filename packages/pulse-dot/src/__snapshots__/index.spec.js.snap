// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders correctly 1`] = `
.c0 {
  position: relative;
}

.c1 {
  border: 3px solid #62bd19;
  width: 25px;
  height: 25px;
  background-color: #62bd19;
  border-radius: 50%;
  position: absolute;
  top: 25px;
  left: 25px;
}

.c2 {
  border: 3px solid #62bd19;
  width: 25px;
  height: 25px;
  border-color: #62bd19;
  border-radius: 50%;
  position: absolute;
  top: 25px;
  left: 25px;
  -webkit-animation: eGhXTC 1.5s ease-out;
  animation: eGhXTC 1.5s ease-out;
  -webkit-animation-iteration-count: infinite;
  animation-iteration-count: infinite;
  opacity: 0;
}

<span
  className="c0"
>
  <span
    className="c1"
  />
  <span
    className="c2"
  />
</span>
`;
